src/lotus/mod.rs

```diff
@@ -43,6 +43,11 @@ pub trait LotusClient {
         msg: MpoolPushMessage,
     ) -> Result<MpoolPushMessageResponseInner>;
 
+    /// Push the unsigned message to memory pool. This will ask the local key store to sign the message.
+    /// In this case, make sure `from` is actually present in the local key store.
```

Comment:
How are we handling the key not being present in the local wallet? We should just abort, but just wondering in case we are sharing enough feedback about the error to users.

Reply:
This one seems need to be handled from `WalletStore`. Error message will be key not exists.
![image](https://github.com/consensus-shipyard/ipc-agent/assets/108330426/456feb5f-f07a-4c33-a236-c952f63dc029)

Reply:
That works, we can keep it as is for now. Thanks!